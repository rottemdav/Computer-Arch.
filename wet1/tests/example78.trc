4 7 12 3 local_history global_tables using_share_lsb
0x80bc N 0x6d0
0x17137db8 T 0x5eda50
0x1641f98 N 0x14dc
0x663117cc N 0xf937c
0x1724 N 0x1b8
0x090c N 0x9009d04
0x1641f98 T 0x14dc
0x6ac T 0x465f24
0xf480 N 0x9e1be6f0
0xf480 T 0x9e1be6f0
0xc01154 T 0xa6eac74
0x9e1b838 N 0xd00
0xe6c T 0xfe4
0x1724 N 0x1b8
0xc01154 N 0xa6eac74
0x848 N 0x0a3efd40
0x80bc N 0x6d0
0x39618 N 0x3b00
0xe6c N 0xfe4
0x848 T 0x0a3efd40
0x17137db8 N 0x5eda50
0x80bc T 0x6d0
0x6ac T 0x465f24
0x39618 N 0x3b00
0xef10 N 0xf09ec4
0x5d97eb0 T 0xcbb1c
0xf480 N 0x9e1be6f0
0x49eb49b8 T 0x7ac
0x1724 T 0x1b8
0x9e1b838 T 0xd00
0xfe6d7c T 0x9313c
0xadd88 N 0x04658
0x314 T 0x1c54
0x17137db8 N 0x5eda50
0xf480 N 0x9e1be6f0
0x9ca86828 N 0x0f6357c
0x49eb49b8 T 0x7ac
0x1724 T 0x1b8
0x6ac T 0x465f24
0x0839fc T 0xc254
0x6ac N 0x465f24
0x5d97eb0 N 0xcbb1c
0xef10 N 0xf09ec4
0xf480 T 0x9e1be6f0
0xf480 T 0x9e1be6f0
0x314 N 0x1c54
0x39618 N 0x3b00
0x314 N 0x1c54
0xe6c T 0xfe4
0x1641f98 T 0x14dc
0x81c T 0xd1dac
0x5d97eb0 N 0xcbb1c
0x6794 T 0x6bc
0x49eb49b8 T 0x7ac
0xc01154 N 0xa6eac74
0xef10 N 0xf09ec4
0xef10 T 0xf09ec4
0x8ba9d8cc N 0xffd2fc
0xadd88 N 0x04658
0x80bc N 0x6d0
0x9ca86828 T 0x0f6357c
0x1724 T 0x1b8
0x314 T 0x1c54
0x9e1b838 T 0xd00
0x848 T 0x0a3efd40
0x80bc T 0x6d0
0x6ac T 0x465f24
0x848 N 0x0a3efd40
0x39618 N 0x3b00
0x663117cc N 0xf937c
0x6ac N 0x465f24
0x848 N 0x0a3efd40
0x80bc T 0x6d0
0xf480 T 0x9e1be6f0
0xe6c N 0xfe4
0xef10 T 0xf09ec4
0x6e4 T 0x3a32aa8
0x9ca86828 N 0x0f6357c
0x314 T 0x1c54
0x663117cc N 0xf937c
0xc01154 N 0xa6eac74
0x1641f98 N 0x14dc
0x1641f98 N 0x14dc
0x314 T 0x1c54
0x17137db8 T 0x5eda50
0x5d97eb0 N 0xcbb1c
0x80bc N 0x6d0
0xfe6d7c T 0x9313c
0x8ba9d8cc T 0xffd2fc
0xe6c N 0xfe4
0xef10 T 0xf09ec4
0x17137db8 N 0x5eda50
0xc01154 N 0xa6eac74
0x090c T 0x9009d04
0x1641f98 T 0x14dc
0x6794 N 0x6bc
0x6ac N 0x465f24
0x8ba9d8cc T 0xffd2fc
0xc01154 N 0xa6eac74
0xc01154 N 0xa6eac74
0x663117cc T 0xf937c
0x39618 N 0x3b00
0x5d97eb0 T 0xcbb1c
0x090c T 0x9009d04
0x6e4 N 0x3a32aa8
0x6794 N 0x6bc
0x6e4 T 0x3a32aa8
0x6794 T 0x6bc
0xadd88 N 0x04658
0x1724 N 0x1b8
0x1724 T 0x1b8
0x663117cc T 0xf937c
0xe6c N 0xfe4
0x80bc N 0x6d0
0x17137db8 N 0x5eda50
0xef10 N 0xf09ec4
0x39618 N 0x3b00
0xef10 N 0xf09ec4
0xfe6d7c T 0x9313c
0x9ca86828 N 0x0f6357c
0x848 T 0x0a3efd40
0x81c N 0xd1dac
0x0839fc N 0xc254
0x9e1b838 T 0xd00
0xfe6d7c N 0x9313c
0x39618 N 0x3b00
0xef10 N 0xf09ec4
0x1641f98 N 0x14dc
0x5d97eb0 N 0xcbb1c
0x663117cc T 0xf937c
0x848 N 0x0a3efd40
0x314 T 0x1c54
0x6ac T 0x465f24
0x848 T 0x0a3efd40
0x1641f98 T 0x14dc
0x0839fc T 0xc254
0x9e1b838 T 0xd00
0xef10 T 0xf09ec4
0x0839fc T 0xc254
0x6ac T 0x465f24
0xef10 T 0xf09ec4