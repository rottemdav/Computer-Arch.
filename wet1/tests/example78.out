0x80bc N 0x80c0
0x17137db8 N 0x17137dbc
0x1641f98 N 0x1641f9c
0x663117cc N 0x663117d0
0x1724 N 0x1728
0x90c N 0x910
0x1641f98 T 0x14dc
0x6ac N 0x6b0
0xf480 N 0xf484
0xf480 T 0x9e1be6f0
0xc01154 N 0xc01158
0x9e1b838 N 0x9e1b83c
0xe6c N 0xe70
0x1724 N 0x1728
0xc01154 N 0xc01158
0x848 N 0x84c
0x80bc N 0x80c0
0x39618 N 0x3961c
0xe6c N 0xe70
0x848 N 0x84c
0x17137db8 N 0x17137dbc
0x80bc N 0x80c0
0x6ac N 0x6b0
0x39618 N 0x3961c
0xef10 N 0xef14
0x5d97eb0 N 0x5d97eb4
0xf480 N 0xf484
0x49eb49b8 N 0x49eb49bc
0x1724 N 0x1728
0x9e1b838 N 0x9e1b83c
0xfe6d7c N 0xfe6d80
0xadd88 N 0xadd8c
0x314 N 0x318
0x17137db8 N 0x17137dbc
0xf480 T 0x9e1be6f0
0x9ca86828 N 0x9ca8682c
0x49eb49b8 N 0x49eb49bc
0x1724 N 0x1728
0x6ac N 0x6b0
0x839fc N 0x83a00
0x6ac N 0x6b0
0x5d97eb0 N 0x5d97eb4
0xef10 N 0xef14
0xf480 N 0xf484
0xf480 T 0x9e1be6f0
0x314 N 0x318
0x39618 N 0x3961c
0x314 T 0x1c54
0xe6c N 0xe70
0x1641f98 N 0x1641f9c
0x81c N 0x820
0x5d97eb0 N 0x5d97eb4
0x6794 N 0x6798
0x49eb49b8 N 0x49eb49bc
0xc01154 N 0xc01158
0xef10 N 0xef14
0xef10 N 0xef14
0x8ba9d8cc N 0x8ba9d8d0
0xadd88 N 0xadd8c
0x80bc N 0x80c0
0x9ca86828 N 0x9ca8682c
0x1724 N 0x1728
0x314 N 0x318
0x9e1b838 N 0x9e1b83c
0x848 N 0x84c
0x80bc N 0x80c0
0x6ac N 0x6b0
0x848 T 0xa3efd40
0x39618 N 0x3961c
0x663117cc N 0x663117d0
0x6ac N 0x6b0
0x848 N 0x84c
0x80bc N 0x80c0
0xf480 N 0xf484
0xe6c N 0xe70
0xef10 N 0xef14
0x6e4 N 0x6e8
0x9ca86828 N 0x9ca8682c
0x314 N 0x318
0x663117cc N 0x663117d0
0xc01154 N 0xc01158
0x1641f98 N 0x1641f9c
0x1641f98 T 0x14dc
0x314 N 0x318
0x17137db8 N 0x17137dbc
0x5d97eb0 N 0x5d97eb4
0x80bc N 0x80c0
0xfe6d7c N 0xfe6d80
0x8ba9d8cc N 0x8ba9d8d0
0xe6c N 0xe70
0xef10 N 0xef14
0x17137db8 T 0x5eda50
0xc01154 N 0xc01158
0x90c N 0x910
0x1641f98 N 0x1641f9c
0x6794 N 0x6798
0x6ac N 0x6b0
0x8ba9d8cc N 0x8ba9d8d0
0xc01154 N 0xc01158
0xc01154 N 0xc01158
0x663117cc N 0x663117d0
0x39618 N 0x3961c
0x5d97eb0 N 0x5d97eb4
0x90c N 0x910
0x6e4 N 0x6e8
0x6794 N 0x6798
0x6e4 N 0x6e8
0x6794 N 0x6798
0xadd88 N 0xadd8c
0x1724 N 0x1728
0x1724 T 0x1b8
0x663117cc N 0x663117d0
0xe6c N 0xe70
0x80bc N 0x80c0
0x17137db8 N 0x17137dbc
0xef10 N 0xef14
0x39618 N 0x3961c
0xef10 T 0xf09ec4
0xfe6d7c N 0xfe6d80
0x9ca86828 N 0x9ca8682c
0x848 N 0x84c
0x81c N 0x820
0x839fc N 0x83a00
0x9e1b838 N 0x9e1b83c
0xfe6d7c N 0xfe6d80
0x39618 N 0x3961c
0xef10 N 0xef14
0x1641f98 N 0x1641f9c
0x5d97eb0 N 0x5d97eb4
0x663117cc N 0x663117d0
0x848 N 0x84c
0x314 N 0x318
0x6ac N 0x6b0
0x848 T 0xa3efd40
0x1641f98 N 0x1641f9c
0x839fc N 0x83a00
0x9e1b838 N 0x9e1b83c
0xef10 N 0xef14
0x839fc T 0xc254
0x6ac N 0x6b0
0xef10 T 0xf09ec4
flush_num: 67, br_num: 141, size: 456b
